kill.c

```diff
@@ -171,15 +182,22 @@ static void userspace_kill(DIR *procdir, int sig, int ignore_oom_score_adj)
 	fclose(stat);
 
 	if(sig != 0)
+	{
 		fprintf(stderr, "Killing process %d %s\n", victim_pid, name);
 
+		char notif_args[200];
+		snprintf(notif_args, 200, "-i dialog-warning 'EarlyOOM' 'Killing process %d %s'", victim_pid, name);
```

Comment:
Please use just "earlyoom" instead of "EarlyOOM" in all messages. Official name is "earlyoom".